### ISSUE TITLE
com.google.fonts/check/036 ots-sanitize with error code (1)

### ISSUE THREAD

**Issue by laerm0** [googlefonts/fontbakery#2377]
Error:

```
WARNING: STAT: Design axis nameID out of range
WARNING: STAT: Axis value nameID out of range
ERROR: GDEF: bad caret value format: 3
ERROR: GDEF: Invalid ligature caret list
ERROR: GDEF: Failed to parse table
Failed to sanitize file!
```

Fonts are Open Sans in development [here](https://github.com/laerm0/opensans/tree/master/variable_ttf).

**Comment by m4rc1e** [comment]
How are you genning the STAT table?

I'm pretty sure we've already reported the bad caret val issue in ots. I think @felipesanches filed a pr to fix it?

**Comment by laerm0** [comment]
I used fontmake, but it was done using Mike's scripts to build Open Sans, which might have complicated things.

**Comment by felipesanches** [comment]
> I'm pretty sure we've already reported the bad caret val issue in ots. I think @felipesanches filed a pr to fix it?

Yes, i did. And it was merged. So soon it won't be an issue anymore.

**Comment by m4rc1e** [comment]
@laerm0 use https://github.com/googlefonts/gftools/blob/master/bin/gftools-fix-vf-meta.py. His script may be hardcoded for the project.

**Comment by laerm0** [comment]
Thanks @m4rc1e !
